### ISSUE TITLE
Torrent list is not showing up

### ISSUE THREAD

**Issue by templarundead** [transmission/transmission#4087]
### What is the issue?

https://user-images.githubusercontent.com/15968502/199679041-cdeb0aba-766b-4929-a29a-c9ad7373cfee.MOV

### Which application of Transmission?

Web client

### Which version of Transmission?

Latest git

**Comment by templarundead** [comment]
Console throw TypeError: e.getPrimaryMimeType().split is not a function. (In 'e.getPrimaryMimeType().split("/",1)', 'e.getPrimaryMimeType().split' is undefined)
Error: {}

**Comment by templarundead** [comment]
The same issue on firefox/windows  and safari/ios, but works perfect on transmission remote gui/windows

I've built transmission with latest git commit with -DENABLE_WEB=ON flag

**Comment by templarundead** [comment]
Console log![image](https://user-images.githubusercontent.com/15968502/199783317-0088aacb-9d9d-41b8-abb2-04367dfa3a55.png)

**Comment by mikedld** [comment]
Since you're on Windows, could be that you've installed both the Qt client and the daemon and are connecting to the wrong session?

**Comment by templarundead** [comment]
> Since you're on Windows, could be that you've installed both the Qt client and the daemon and are connecting to the wrong session?

No! The transmission-daemon is located on my sbc/ubuntu server to which I connect via web client ios/windows or remote gui/windows

**Comment by templarundead** [comment]
> Sorry, didn't see your comment where you say you're building it yourself. Still, are you sure there's no other session (with no torrents) that you might be connecting to?

Only one session. Here screenshot from Windows

![image](https://user-images.githubusercontent.com/15968502/199790462-e225a854-3cc3-4c2b-803a-5c92db6fe861.png)
